bpf/complexity-tests/510/bpf_host/9.txt

```diff
@@ -0,0 +1,18 @@
+-DENABLE_EGRESS_GATEWAY=1
+-DENABLE_HOST_ROUTING=1
+-DENABLE_ICMP_RULE=1
+-DENABLE_IDENTITY_MARK=1
+-DENABLE_IPV4=1
+-DENABLE_IPV4_FRAGMENTS=1
+-DENABLE_IPV6=1
+-DENABLE_IPV6_FRAGMENTS=1
+-DENABLE_L7_LB=1
+-DENABLE_MASQUERADE_IPV4=1
+-DENABLE_MASQUERADE_IPV6=1
+-DENABLE_NODEPORT=1
+-DENABLE_SERVICE_PROTOCOL_DIFFERENTIATION=1
+-DENABLE_SESSION_AFFINITY=1
+-DENABLE_SOCKET_LB_FULL=1
+-DENABLE_SOCKET_LB_PEER=1
+-DENABLE_SRC_RANGE_CHECK=1
+-DENCAP_IFINDEX=1
```

Comment:
I believe the goal is to match the config from the sysdump at https://github.com/cilium/cilium/issues/40522.

If so, you'll want to add:
```txt
-DALLOW_ICMP_FRAG_NEEDED=1
-DCT_REPORT_FLAGS=0x00ff
-DSERVICE_NO_BACKEND_RESPONSE=1
-DTUNNEL_MODE=1
```
which can all have an impact on complexity AFAICS.